erts/etc/win32/cygwin_tools/reg_query.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/sh
+BAT_FILE=/tmp/w$$.bat
+if [ -z "$1" -o -z "$2" ]; then
+    echo "Usage:" "$0" '<key> <valuename>'
+    exit 1
+fi
+BACKED=`echo "$1" | sed 's,/,\\\\,g'`
+# We need to get the 64bit part of the registry, hence we need to execute 
+# a 64bit reg.exe, but c:\windows\system32 is redirected to 32bit versions
+# if we ate in the 32bit virtual environment, why we need to use the 

```

Comment:
Small typo 
- ate -> are
- why -> so (or -> that's why)

Reply:
Thanks fixed